packages/api/src/routes/userFarmRoute.js

```diff
@@ -98,7 +98,7 @@ router.post(
   '/invite/farm/:farm_id/user/:user_id',
   hasFarmAccess({ params: 'farm_id' }),
   checkScope(['edit:users']),
-  userFarmController.upgradePseudoUser(),
+  userFarmController.patchPseudoUserEmail(),
```

Comment:
I think the name here was changed to upgradePseudoUser. @seinnlee can you take a look at the code here that involves your pseudo user fix?